logos/src/logos/dbutils/dbrequest.py

```
@@ -1,4 +1,4 @@
-from typing import Optional, Union
+from typing import List, Optional, Union
```

Flake8 (7.3.0)

[error] 'typing.Optional' imported but unused

(F401)